src/etc/inc/interfaces.inc

```diff
@@ -5403,7 +5403,7 @@ function DHCP6_Config_File_Advanced($interface, $wancfg, $wanif) {
 		$key_info_statement .= "\trealm \"{$wancfg['adv_dhcp6_key_info_statement_realm']}\";\n";
 		$key_info_statement .= "\tkeyid {$wancfg['adv_dhcp6_key_info_statement_keyid']};\n";
 		$key_info_statement .= "\tsecret \"{$wancfg['adv_dhcp6_key_info_statement_secret']}\";\n";
-		if (preg_match("/((([0-9]{4}-)?[0-9]{2}[0-9]{2} )?[0-9]{2}:[0-9]{2})||(foreever)/", $wancfg['adv_dhcp6_key_info_statement_expire'])) {
+		if (preg_match("/((([0-9]{4}-)?[0-9]{2}[0-9]{2} )?[0-9]{2}:[0-9]{2})||(fore?ever)/", $wancfg['adv_dhcp6_key_info_statement_expire'])) {
```

Comment:
This is the most exciting part of the PR. `foreever` is a very strange token.

Reply:
Bug, it should just be `forever` as specified here:
https://www.freebsd.org/cgi/man.cgi?query=dhcp6c.conf